app/views/guests/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @guest.name %></h1>
+<ul><% @rooms.each do |room| %>
+  <li><%= room.suite %>- Rate: <%= room.rate %>, Hotel: <%= room.hotel.name %></li>
```

Comment:
This doesn't necessarily match what you have in your test, where the test just says the values not related to the keys ("Rate: ", "Hotel:" ). Be careful that your feature tests echo what the view is exactly.